digdag-ui/package.json

```diff
@@ -32,7 +32,7 @@
     "moment-duration-format": "1.3.0",
     "pako": "1.0.5",
     "path": "0.12.7",
-    "popper.js": "1.14.4",
+    "popper.js": "^1.15.0",
```

Comment:
@szyn Is it safe?  Does it accept the version between 1.15.0 and 2.0.0?

Reply:
@yoyama As my understanding, basically it is safe because based on Semantic Versioning. Yes, it accept the version like this `1.15.0 <= version < 2.0.0`.

Reply:
I see. Thanks!